css/_details.scss

```diff
@@ -310,6 +310,13 @@ detailsitem .icon-delete {
 	background-color: transparent;
 	border: none;
 	opacity: .2;
+	visibility: hidden;
+}
+
+detailsitem input:hover + .icon-delete,
```

Comment:
Don't you want to use the hover state on the detailsitem element directly?
This would be a solution for the address issue :)

Reply:
Good enhancement, however there's no :active state for detailsitem, so it won't show when you are in a field of the address?

Reply:
Good call! :/
But I really would like to avoid js!

Reply:
Alright, I fixed it, check it out! :) Is also now accessible via keyboard cause I switched from `visibility: hidden;` to `opacity: 0;` :rocket: